docs/source/configuration/configuration-run.rst

```diff
@@ -477,10 +477,10 @@ Link Prediction Task
     - Yaml: ``gamma: 10.0``
     - Argument: ``--gamma 10.0``
     - Default value: ``12.0``
-- **lp_loss_func**: Link prediction loss function. Builtin loss functions include ``cross_entropy`` and ``logsigmoid``.
```

Comment:
logsigmoid is gone? is this API change?

Reply:
The is no code of logsigmoid. I think we mistakenly add this in our doc.